modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -106,7 +92,7 @@ Where `<num>` will be the worker number.
 +
 [source,bash]
 ----
-[kni@provisioner ~]$ oc -n openshift-machine-api get bmh openshift-worker-<num>
```

Comment:
Per the Supplementary Style Guide, [user-replaced values](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values) within code blocks need the [subs="+quotes"] attribute to apply text formatting.